### ISSUE TITLE
Mask RCNN - Normalization - mean & std

### ISSUE THREAD

**Issue by yc0619** [open-mmlab/mmdetection#4801]
Hi,
I have trained my own dataset on Mask RCNN. Since my GPU is limited, i want to set a smaller resize by training and validation.
So, at the train pipe line, the resized image = 512, 512 (default 1333, 800)
At the val pipe line, the resized image = 695, 695.
I would like to know how can i adjust the "mean" and "std" by "img_norm_config". Is there a tutor for it?
thanks!

**Comment by hhaAndroid** [comment]
You can compare the two options：

1. Instead of calculating the mean and std, it is just using the imagenet value in the default config. Then use coco pre-training weights to fintune.
2. Iterate over the entire image dataset, then recalculate the mean and std. Finally use coco pre-training weights to finetune.

**Comment by ZwwWayne** [comment]
The image mean and std are inherits from the backbone model, which is pre-trained on  ImageNet.
